.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
dependabot is incredibly annoying, I think it'd be better to do something like this: https://github.com/kubernetes/cloud-provider-aws/blob/master/.github/workflows/update-deps.yaml

Reply:
Seems vpc-cni also use the dependabot. https://github.com/aws/amazon-vpc-cni-k8s/blob/b0ff9b904a54a9bec276b7510a92d14ea6ab2d90/.github/dependabot.yml, when you say annoying, is it sending notification frequently?

while the workflow solution is already used in cloud-provider-aws repo, just double confirm the plugins used like peter-evans/create-pull-request, are approved by AppSec?